int32 bottle_left
---
bool success
string message
